src/libraries/Microsoft.PowerFx.Core/Entities/External/IExternalEntityScope.cs

```diff
@@ -18,6 +18,15 @@ internal interface IExternalEntityScope
 
         bool TryGetEntity<T>(DName currentEntityEntityName, out T externalEntity)
             where T : class, IExternalEntity;
+
+        /// <summary>
+        /// Checks if the given name is available or not.
+        /// <para>This is used in Tokenization to determine if namespace of the dotted name can be hidden or not.</para>
```

Comment:
Note: not a namespace. Perhaps: "Determine if LHS of the dotted name ..."

Reply:
Done